client/app/main/student/student.controller.js

```diff
@@ -2,18 +2,8 @@
 
 var app = angular.module('app');
 
-function StudentCtrl($scope, $stateParams, Intervention, Modal, Outreach,
-  Student, toastr) {
-
+function StudentCtrl($scope, $state, $stateParams, Student, toastr) {
   $scope.student = Student.get({id: $stateParams.id});
```

Comment:
Shouldn't modal be passed as an argument since it's still being used in the controller?